File: c/soma_vetor.c
#include <stdio.h>

int main(){
    int n;
    double soma, media;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vetor[n];

    for (int i=0; i<n; i++) {
        printf("Digite um numero: ");
        scanf("%lf", &vetor[i]);
    }

	soma = 0;
    for (int i=0; i<n; i++) {
        soma = soma + vetor[i];
    }

    media = soma / n;

	printf("VALORES = ");

    for (int i=0; i<n; i++) {
        printf("%.1lf  ", vetor[i]);
    }

    printf("\nSOMA = %.2lf\n", soma);
	printf("MEDIA = %.2lf\n", media);

    return 0;
}

File: java/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
	    double soma, media;

	    System.out.print("Quantos numeros voce vai digitar? ");
	    n = sc.nextInt();

	    double[] vetor = new double[n];

	    for (int i=0; i<n; i++) {
	    	System.out.print("Digite um numero: ");
	        vetor[i] = sc.nextDouble();
	    }

		soma = 0;
	    for (int i=0; i<n; i++) {
	        soma = soma + vetor[i];
	    }

	    media = soma / n;

		System.out.print("VALORES = ");

	    for (int i=0; i<n; i++) {
	    	System.out.printf("%.1f  ", vetor[i]);
	    }

	    System.out.printf("\nSOMA = %.2f\n", soma);
	    System.out.printf("MEDIA = %.2f\n", media);

		sc.close();
	}
}
